### DISCUSSION TITLE
Firefox ESR Background Updates

### DISCUSSION THREAD

**Discussion by vossni** [mozilla/policy-templates#913] [Q&A] [answered]
Hello,

Firefox has supported automatic background updates (https://support.mozilla.org/en-US/kb/firefox-enterprise-90-release-notes#w_general) since version 90. As far as I know, this is not yet supported by the Firefox ESR. Does anyone know if with the change of Firefox ESR to version 102, this feature will also be supported by Firefox ESR? If so, will the corresponding policy (BackgroundAppUpdate) also be applied to Firefox ESR 102?

Best regards

**Comment by mkaply** [comment, marked as answer]
This was definitely supported in the ESR (all the code was backported). I don't know why some people had trouble with it.
But it's definitely in Firefox 102 and the 102 ESR

**Reply by vossni** [reply]
Thanks for the reply.
